x/gasless/types/errors.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	sdkerrors "cosmossdk.io/errors"
+)
+
+// DONTCOVER
+
+var (
+	ErrorUnknownProposalType       = sdkerrors.Register(ModuleName, 10000, "unknown proposal type")
+	ErrorInvalidrequest            = sdkerrors.Register(ModuleName, 10001, "invalid request")
```

Comment:
Please remove and use sdkerrors invalid req

Reply:
fixed